src/devtools/WebAudioGraphIntegrator.ts

```diff
@@ -603,11 +693,14 @@ Context was likely cleaned up during request for realtime data.
   ) => {
     if (debuggerDetached.reason === 'target_closed') {
       console.debug(
-        `Checking if tracked Audio Contexts (${Object.keys(contexts)
-          .map((contextId) => contextId.slice(-6))
-          .join(
-            ', ',
-          )}) exist after debugger detached because target was closed.`,
+        '[' +
```

Comment:
Perhaps you can define this as a function somewhere?

```js
const getTimestampAsString = () => {
  return `[${performance.now().toFixed(2)}]`;
}
```

Then the whole thing can be:
```js
`${getTimestampAsString()} Checking if tracked AudioContexts....`
```